ci/jenkins/Jenkinsfile.j2

```diff
@@ -103,6 +103,19 @@ s3_prefix = "tvm-jenkins-artifacts-prod/tvm/${env.BRANCH_NAME}/${env.BUILD_NUMBE
 {% set aws_default_region = "us-west-2" %}
 {% set aws_ecr_url = "dkr.ecr." + aws_default_region + ".amazonaws.com" %}
 
+// Any scripts that run on the bare host and not inside a Docker container
+// (especially those that access secrets) should be in this list so that only
+// trusted versions of these files are used in CI
+{% set trusted_files = [
```

Comment:
should this just be a glob for simplicity?

Reply:
we should do some restructuring (to split test running scripts up from these kinds of scripts into their own folder) but that does make more sense.

Reply:
#12609